script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_10.txt

```diff
@@ -27,12 +27,12 @@ It's terribly familiar. I feel like I saw something similar in my dream at least
 <Page8>
 Arcueid walks up to me as if nothing had happened.
 <Page9>
-"........Well, I'm alive, at least."
-The truth is that I feel like I'm dying so much that it's painful just to speak.
+"―――Well, I'm alive, at least."
+I mean it feels like I'm dying, just speaking is painful.
```

Comment:
it feels like I'm dying; just speaking is painful.

Reply:
Would changing it to:
"To tell the truth it feels like I'm dying, even just talking brings pain"

Reply:
'I mean' is a fairly fluid colloquialism that can be used to mean a number of things.

To be honest, it feels like I'm dying; even speaking is painful.